jdaviz/configs/imviz/plugins/orientation/orientation.py

```diff
@@ -368,6 +368,19 @@ def _on_refdata_change(self, msg={}):
             elif not len(viewer.data()):
                 self.link_type_selected = link_type_msg_to_trait['pixels']
 
+        # we never want to highlight subsets of pixels within WCS-only layers,
+        # so if this layer is an ImageSubsetLayerState on a WCS-only layer,
+        # ensure that it is never visible:
+        if hasattr(self, 'viewer'):

```

Comment:
Why does this have to be a different `if` if it is the same check as L358 above?

Reply:
Because this block should come after the previous lines that are above this `if`, but below the upper `if`.